### PR TITLE
Add a button to reset the profile image(fixes 4551)

### PR DESCRIPTION
**Pull request by ScottHuangNYU**

### CODE REVIEW

File: src/app/users/users-update/users-update.component.ts [hunk 1]
```diff
@@ -168,6 +174,11 @@ export class UsersUpdateComponent implements OnInit {
     this.uploadImage = false;
   }
 
+  ResetImage() {
```
Comment (paulbert) [inline src/app/users/users-update/users-update.component.ts]: Methods should start with a lowercase letter.  Only classes should start with an uppercase letter.

**Comment by paulbert** [comment]
The suggestion here will get you halfway: https://github.com/harryy2510/ngx-img/issues/14

You will also need to set `mode = 'crop'`

**Comment by paulbert** [comment]
@ScottHuangNYU By `mode = 'crop'` I mean the example I linked to would change to:

```
<ngx-img #imageuploader (onSelect)="onSelect($event)" (onReset)="reset()" [config]="config"></ngx-img>

@ViewChild('imageuploader') public imageUploader: NgxImgComponent;

public ngAfterViewInit(): void {
    this.imageUploader.hasPreview = true;
    this.imageUploader.imgSrc = this.initialImageUrl;
    this.imageUploader.mode = 'crop';
}
```

File: src/app/users/users-update/users-update.component.ts [hunk 2]
```diff
@@ -177,4 +179,10 @@ export class UsersUpdateComponent implements OnInit {
     });
   }
 
+  ngAfterViewInit(): void {
+    this.imageUploader.hasPreview = true;
+    this.imageUploader.imgSrc = 'assets/image.png';
```
Comment (paulbert) [inline src/app/users/users-update/users-update.component.ts]: @ScottHuangNYU `this.imageUploader.imgSrc` should be assigned to the user's profile image src (if they have one)

**Comment by paulbert** [comment]
Putting this one "On Hold".  When the cropper initializes it automatically cuts the image down by 80%, so every time the member edits their profile their image will change unless they manually extend the crop to select the full image.

From what I can tell, the `ngx-img` library does not include the option to change that, but `cropper` which is used by `ngx-img` does.  Might make an issue to change that.

**Comment by Mutugiii** [comment]
Resolved in #7378

**Pull request closed without merging**